tls/client-tls-nonblocking.c

```diff
@@ -160,17 +246,27 @@ int main(int argc, char** argv)
             continue;
         }
         fprintf(stderr, "ERROR: failed to write\n");
-        return -1;
+        ret = -1;
```

Comment:
The example is not exiting after the write. Also it needs to read the echo from the server and then shutdown.